tests/baselines/reference/classSideInheritance1.errors.txt

```diff
@@ -16,9 +16,9 @@ tests/cases/compiler/classSideInheritance1.ts(13,3): error TS2339: Property 'bar
     var c: C2;
     a.bar(); // static off an instance - should be an error
       ~~~
-!!! error TS2339: Property 'bar' does not exist on type 'A'.
+!!! error TS2576: Property 'bar' is declared static and can not be called from an instance

```

Comment:
I would leave off the elaboration (and 'declared') and instead mention the type name: "Property 'bar' is a static member of type 'A'."